aspnet/web-forms/videos/net-4/routing/how-do-i-use-routing-with-aspnet-web-forms.md

```diff
@@ -15,7 +15,7 @@ by [Chris Pels](https://twitter.com/chrispels)
 
 In this video Chris Pels shows how to implement routing for Web Forms in ASP.NET 4. First, the concept of routing a URL is compared to mapping the URL to a physical file in the site. Then, a sample route for a URL is defined in the global.asax file Application\_Start event handler. The route contains a parameterized value that the user can enter in the URL. A sample page is then created and the route parameter value is extracted in the Page\_Load event handler. Next, a second route is defined that has multiple parameters and routes to the same page as the initial route. The Page\_Load event handler is expanded to extract the additional route parameter value and display different information depending upon what values have been passed to the page.
 
-[&#9654; Watch video (15 minutes)](https://channel9.msdn.com/Blogs/ASP-NET-Site-Videos/how-do-i-use-routing-with-aspnet-web-forms)
+[&#9654; Watch video (15 minutes)](https://learn.microsoft.com/shows/asp-net-site-videos/how-do-i-use-routing-aspnet-web-forms)
```

Comment:
This video has a valid relative link but it is not possible to get the build to work with it.  I verified the path.  So I ended up having to use a complete URL path.